tests/bluetooth/tester/src/btp/btp_gap.h

```diff
@@ -599,3 +600,32 @@ int tester_gap_padv_stop(struct bt_le_ext_adv *ext_adv);
 int tester_gap_padv_create_sync(struct bt_le_per_adv_sync_param *create_params);
 int tester_gap_padv_stop_sync(void);
 #endif /* defined(CONFIG_BT_EXT_ADV) */
+
+#if defined(CONFIG_BT_EAD)
+#define BTP_GAP_EAD_SET_KEY_MATERIAL 0x31
+#define BTP_GAP_EAD_CREATE_ADV_DATA  0x32
```

Comment:
this should be name BTP_GAP_EAD_ENCRYPT_ADV_DATA (and not "create" to better match btp spec name)